base_external_api_ftp/models/apicli_connection_ftp.py

```diff
@@ -86,7 +86,8 @@ def cron_download_ftp_files(self, subdirectory="/", conn_code=None):
                 ) as sftp:
                     conn._download_each_file(subdirectory, sftp)
 
-        self.env["apicli.message"].process_messages()
+        messages = self.env["apicli.message"].search([("state", "=", "todo")])
```

Comment:
@dreispt we already filtered message in **process_messages**.

see here: https://github.com/OCA/server-backend/pull/151/files#diff-fc037587474562c4b7f5bad821396cd3b4908f76a2d1a9e8a9c6540d2a16c179R46

Reply:
The process_messages could also be called directly. In fact, could you add that ability to the “Action” menu of the Stock Picking please?